keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -741,14 +821,23 @@ levels and a proposal for their new configuration.
 | global-default  |  20 |  50% | none |
 | catch-all       |   5 |   0% | none |
 
-Each non-exempt priority level `i` has two concurrency limits: its
+The following table shows the `ExemptPriorityLevelConfiguration`
+introduced for the exempt priority levels in the default
+configuration.
+
+| Name | Nominal Shares | Lendable |
+| ---- | -------------- | -------- |
+| exempt |           30 | 50%      |
```

Comment:
This needs to be updated, given we actually changed this :)

Reply:
Oops, thanks.  Fixed in latest revision.